source/deploy/server/deploy-kubernetes.rst

```diff
@@ -71,4 +71,4 @@ For Volume Mounts, the following options use Kubernetes Volume Mounting to provi
 - NFS
 - AzureBlob
 
-Not all types of `Kubernetes persistent volumes <https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes>`_ have been tested with Mattermost, and some may have limitations or specific configurations that may require additional setup to ensure proper permissions and access. We recommend system admins review documentation for their preferred persistent volume types and test to ensure compatibility with Mattermost.
\ No newline at end of file
+Not all types of `Kubernetes persistent volumes <https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes>`_ have been tested with Mattermost, and some may have limitations or specific configurations that may require additional setup to ensure proper permissions and access. We recommend system admins review documentation for their preferred persistent volume types and test to ensure compatibility with Mattermost.
```

Comment:
@cwarnermm - I have no idea why this is here. I didn't change it but it also doesn't seem like a change so i haven't reverted it all.

Reply:
That content is the last paragraph of an FAQ at the bottom of the page and is intended to be there (recently added).

Reply:
Sorry i meant the changed line, since i didn't touch it.